data/report_templates/project_summary.html

```diff
@@ -396,42 +410,32 @@
   <div class="clearfix"></div>
-  National Genomics Infrastructure - $title$
+  National Genomics Infrastructure - {{ get(meta.title) }}
```

Comment:
Is this the regular Python function or something special with Jinja2? If it's regular Python, does it need to be `meta.get('title')` or `getattr(meta, 'title')` instead?

Reply:
Its a function I defined in jinja as `jinja2_env.globals['get'] = lambda x: x[0]`. I guess I could just do x[0] instead of using a function.

Reply:
Ah ok. Why is it x[0] in the first place?

Reply:
The Markdown meta extension returns the metadata values as lists.